package.json

```diff
@@ -62,25 +62,34 @@
         "@graphql-eslint/eslint-plugin": "^3.20.1"
     },
     "devDependencies": {
+        "@commitlint/cli": "^19.3.0",
+        "@commitlint/config-conventional": "^19.2.2",
         "@eslint/js": "^9.2.0",
         "@jest/globals": "^29.7.0",
-        "@types/jest": "^29.5.12",
+        "@semantic-release/changelog": "^6.0.3",
+        "@semantic-release/exec": "^6.0.3",
+        "@semantic-release/git": "^10.0.1",
+        "@semantic-release/npm": "^12.0.1",
         "@types/eslint": "^8.56.10",
         "@types/estree": "^1.0.5",
+        "@types/jest": "^29.5.12",
         "@typescript-eslint/eslint-plugin": "^7.8.0",
         "@typescript-eslint/parser": "^7.8.0",
         "@typescript-eslint/rule-tester": "^7.8.0",
         "@typescript-eslint/utils": "^7.8.0",
+        "conventional-changelog-conventionalcommits": "^8.0.0",
```

Comment:
Cross reference Jason Schroeder's recent fix on `release-notes-generator` update. Version `^8.0.0` on `conventional-changelog-conventionalcommits' will not cause issue. So we don't need to do anything.